updates (#34710)
dotnet/docs: docs/azure/key-azure-services.md

| Icon | Service | Description |
|:----:|:--------|:------------|
| ![App Service Icon](./media/app-services.svg) | **Azure App Service** | Azure App Service is a fully managed platform for hosting web applications and APIs in Azure.  It features automatic load balancing and auto-scaling in a highly available environment.  You pay only for the compute resources you use and free tiers are available. |

> | ![Container Apps Icon](./media/container-apps.svg) | **Azure Container Apps** | Azure Container Apps is a fully managed environment that enables you to run microservices and containerized applications on a serverless platform. Applications built on Container Apps provide scalability, resiliency and other advantages of complex contaienr orchestrators while leaving behind many of the complexities.  |

Reviewer: ```suggestion
| ![Container Apps Icon](./media/container-apps.svg) | **Azure Container Apps** | Azure Container Apps are fully managed environments that enable you to run microservices and containerized applications on a serverless platform. Applications built on Container Apps provide scalability, resiliency, and other advantages of complex container orchestrators while leaving behind many of the complexities.  |
```
